en/04_Changelogs/5.0.0.md

```diff
@@ -203,6 +204,47 @@ their aliases (applied by setting a string key for the array) were being ignored
 - PHPUnit 5.7 is no longer supported. It is recommended that you ensure your tests are running with PHPUnit 9 in your Silverstripe CMS 4 project before upgrading to Silverstripe CMS 5. There is [documentation for the upgrade process here](https://docs.silverstripe.org/en/4/upgrading/phpunit9/).
 - `masterminds/html5` is now used to parse HTML content instead of `DomDocument::loadHTML()`. This may cause slight changes in rendered content if the content being saved contains invalid HTML. The optional module `silverstripe/html5` is no longer required for rendering HTML 5 content.
 
+### URLs generated by Silverstripe CMS don't include a trailing slash by default {#trailing-slash}
+
+The URLs generated by Silverstripe CMS have previously been inconsistent as to whether a trailing slash is included or not. This lead to the creation of [at least one module](https://github.com/axllent/silverstripe-trailing-slash) which normalises the trailing slash at the end of URLs.
+
+By default, all URLs created by Silverstripe will now _not_ end with a slash. You can configure this to instead explicitly add a trailing slash to all URLs with the following YML configuration:
+
+```yml
+SilverStripe\Control\Controller:
+    add_trailing_slash: true
+```
+
+[warning]
+Because this can be controlled with configuration, it is best practice to avoid explicitly expecting a trailing slash to either be present or be omitted.
+
+In PHP you can use methods like [Controller::join_links()](api:SilverStripe\Control\Controller::join_links()) or [Controller::normaliseTrailingSlash()](api:SilverStripe\Control\Controller::normaliseTrailingSlash()). In JavaScript in your CMS customisations, we recommend using using the `joinUrlPaths()` utility function from [@silverstripe/webpack-config](https://www.npmjs.com/package/@silverstripe/webpack-config). In your templates, you should use appropriate methods from your controller or model such as [SiteTree::Link()](api:SilverStripe\CMS\Model\SiteTree::Link()) which uses `Controller::join_links()` under the hood.
```

Comment:
joinUrlPaths() is in admin, not webpack-config https://github.com/silverstripe/silverstripe-admin/pull/1438/files#diff-3f5e793718f99fa848e4676d4d27c8f4a5cf62ca3b6f291168d66facefcd49f5R1

Reply:
Yes. But it cannot be used without webpack-config.

Reply:
> `joinUrlPaths()` utility function from [@silverstripe/webpack-config]

Well, it's not FROM webpack-config, so we'll need to change this sentence to something else.  Is webpack-config even relevant? Wouldn't webpack-config just be available for anyone working in the cms since it's in node module.  It's it just a case of importing something that's always there?

Reply:
It is technically exposed as a global, so you could go `ssUrlLib.joinUrlPaths()` - but the way that it's exposed is just an implementation detail of webpack expose-loader and webpack externals, and the name `ssUrlLib` is somewhat arbitrary, so I wouldn't want to encourage people to access it that way.
Instead, we should encourage people to use `webpack-config` when they want to use our exposed libraries or components. I'll change to wording to be clearer.

Reply:
All you need is `import { joinUrlPaths } from 'lib/urls';` right?  That's all people need to know in order to use this on their module?  Or do you explicitly need webpack-config as a devDependency?

Reply:
`lib/urls` comes from admin, and unless you explicitly set your webpack to grab from admin, it will not resolve. The supported and recommended way of doing this is to use webpack-config.

Reply:
I've made it clear it's part of `silverstripe/admin`, but retained the recommendation of using `@silverstripe/webpack-config`. I've also indicated _how_ to import it, which I hadn't realised I'd omitted.